test/status-handler.test.js

```diff
@@ -19,7 +19,26 @@ describe('Status-Handler', function () {
     before(() => {
       const data = helpers.connectDatastar({ mock: true }, mocks.datastar());
       status = new StatusHandler({
-        models: models(data)
+        models: models(data),
+        webhooks: {
+          whatever: [
+            'http://example.com/webhooks'
+          ]
+        }
+      });
+    });
+
+    describe('_getPackageName', function () {
+      it('should return the package name', function () {
+        const pkg = status._getPackageName(fixtures.singleQueued);
+        assume(pkg).equals('whatever');
+      });
+    });
+
+    describe('_shouldSendWebhook', function () {
+      it('should check if a package has registered endpoints', function () {
+        assume(status._shouldSendWebhook('whatever')).equals(true);
+        assume(status._shouldSendWebhook('whatever2')).equals(false);
       });
```

Comment:
you need more tests to cover:

1. Multiple webhooks
1. Actually sending a webhook
1. Dealing with a webhook where:
    1. The service is down
    1. The webhook returns errors
    1. The webhook is slow

Reply:
Yeah, look at my PR title. It was WIP. I just pushed the tests.

Reply:
Of course. PR was still WIP when you reviewed. Just pushed the e2e tests.

Reply:
At the end, I think from an e2e prospective we are interested in ensuring that everything works if webhook request succeed, and if it fails, it does not stop sending the other web-hooks nor it crashed the app.